src/blog/2022/09/flowforge-09-released.md

```diff
@@ -0,0 +1,86 @@
+video: BBNzFInc_2E
+---
+
+Suspend your projects when you don't need them, forgot your username? login with your email instead, and introducing Team Types
+<!--more-->
+
+Keep reading for the details of whats in this release our you can watch our 1 minute roundup video of the new release above. 
+
+We're pleased to announce version 0.9 is now available. The next release of the FlowForge application containing new features, a number of improvements, and bug fixes. Keep reading for a promotion code to get your first month free on FlowForge Cloud. 
+
+## Features
+[Suspend Projects](https://github.com/flowforge/flowforge/issues/893)
+Sometimes you want to put a project to one side for a while, maybe your development has stalled or you're waiting on something external to be ready. Perhaps you don't need it to be running all the time. With the 0.8 release we've made some changed to how projects are run. You can now suspend a project, what this means is that the container in which your project runs is removed and nothing is running, Your flows  are safely stored in our database but the project isn't consuming any resources. This means that in FlowForge Cloud we also suspend billing you for it.
+Your project will be there ready to start back up when you need it with just one click, the name and hence the URL are still reserved for you.
+Remember that any context data or anything written to the filesystem will not persist through a restart or a suspend of a project.
+As a result of this change we've also removed the facility to stop a project, Stopping only halted the Node-RED process it didn't remove the resources so was of very limited value, as of Node-RED 3.0 you can stop the flows from executing while still being able to make changes which is a better experience.
+You can still restart the Node-RED process from the Forge app as before for example when you have updated a package in your flows.
+
+[Team Types](https://github.com/flowforge/flowforge/issues/733)
+We've introduced another concept into the platform with this release. Team Types will allow us to offer more advanced features to teams on FlowForge Cloud, You won't see much difference in this release but it allows us to build on in future releases.
+
+[PostHog Analytics](https://github.com/flowforge/flowforge/issues/695)
+We've added analytics to the platform, we're using [PostHog](https://posthog.com/) for this as they share our ethos of open source and self hosting, If you're running your own platform.
+For FlowForge Cloud users the data is sent to our PostHog account, but if you're running your own instance then it won't send analytics to us, you are free to configure your own PostHog service though to receive the data yourself.
+
+
+[Login with email](https://github.com/flowforge/flowforge/issues/856)
+A common problem that we've seen from users is trying to login with their email address instead of their username, as of 0.8 you can now enter either at the login screen.
```

Comment:
```suggestion
A common problem that we've seen from users is trying to login with their email address instead of their username. As of 0.9 you can now enter either at the login screen.
```